SoC-2016-Microprojects.md

```diff
@@ -19,6 +19,18 @@ libgit2 list of
 projects](https://github.com/libgit2/libgit2/blob/master/PROJECTS.md)
 rather than the list below.*
 
+Git development is based on sending successive versions of patchs or
+patch series to the mailing list until they are considered good and
+correct by the reviewers and Junio Hamano, the maintainer, who will
+merge them. This process usually takes quite some time. By sending
+drafts of your patches for your microproject, to the
```

Comment:
I think this comma is extraneous. Might also be easier to parse as `sending drafts of your microproject patches to the mailing list...` to reduce the recursion level of prepositional phrases.